assets/blocks/course-progress-block/course-progress-edit.js

```diff
@@ -66,10 +66,7 @@ export const CourseProgressEdit = ( props ) => {
 		},
 	};
 	const barWrapperAttributes = {
-		className: classnames(
-			'wp-block-sensei-lms-progress-bar',
-			barBackgroundColor?.class
-		),
+		className: classnames( barBackgroundColor?.class ),
```

Comment:
We can get rid of the call to `classnames` since we're just dealing with a single class now.

Reply:
Also, I found an instance of this class [here](https://github.com/Automattic/sensei/blob/94c8d1767999d0a73b6cfd8770d83c3597193c06/assets/blocks/learner-courses-block/learner-courses-editor.scss#L35), so I guess we'll need to update the class used there.

Reply:
Yeah, that is the new class, but it wasn't used on the places where we used ```.wp-block-sensei-lms-progress-bar``` also I would not rename it since it's not in shared folder

Reply:
Will remove classnames